pom.xml

```diff
@@ -0,0 +1,249 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@dblock the project uses Apache Maven, if you have strong preference towards Gradle, I could rework it, please let me know

Reply:
Just curious, what was the reason for using Maven? Generally consistency is nice, and we seem to use Gradle for most (I think?) other Java repos in the project.

Reply:
Mostly simplicity & stability of Maven build model / tooling, but for consistency Gradle would be preferred

Reply:
@andrross moved to Gradle